pkgs/development/python-modules/seafobj/setuptools.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/setup.py b/setup.py
+new file mode 100644
+index 0000000..898cbe7
+--- /dev/null
++++ b/setup.py
+@@ -0,0 +1,18 @@
++from setuptools import setup, find_packages
++
++__version__ = '6.2.10'
++
++
++setup(name='seafobj',
++      version=__version__,
++      license='BSD',
```

Comment:
I don't love this method to add the setup.py, because it means I'd have to maintain the patch too. Is there a better way to package a python library without a setuppy?